<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>weeklyScheduleEditDialog</class>
 <widget class="QDialog" name="weeklyScheduleEditDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>501</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="fieldGrowthPolicy">
     <enum>QFormLayout::FieldsStayAtSizeHint</enum>
    </property>
    <item row="1" column="1">
     <widget class="QLineEdit" name="descriptionLineEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Start day of week</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="startWeekdayComboBox">
      <item>
       <property name="text">
        <string>Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thirsday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Friday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sunday</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>End day of week</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="endWeekdayComboBox">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <item>
       <property name="text">
        <string>Monday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Tuesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Wednesday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Thirsday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Friday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Saturday</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sunday</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>550</y>
     <width>193</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="singleFileRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>Play single file once</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="musicFolderRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>401</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Play all files in folder with rotation during time (music)</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>501</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="singleFileWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>90</height>
       </size>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>481</width>
         <height>81</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Time</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sound file</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTimeEdit" name="timeEdit">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="displayFormat">
           <string>hh:mm:ss</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="fileNameLineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="chooseFilePushButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="playSoundPushButton">
            <property name="text">
             <string>Play</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="musicFolderWidget" native="true">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>70</height>
       </size>
      </property>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Start time</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="startTimeEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>0</y>
         <width>118</width>
         <height>26</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>0</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>End time</string>
       </property>
      </widget>
      <widget class="QTimeEdit" name="timeEdit_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>0</y>
         <width>118</width>
         <height>26</height>
        </rect>
       </property>
       <property name="displayFormat">
        <string>hh:mm:ss</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="folderNameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>38</y>
         <width>211</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
      <widget class="QPushButton" name="chooseFolderPushButton">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>38</y>
         <width>93</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>weeklyScheduleEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>weeklyScheduleEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleFileRadioButton</sender>
   <signal>pressed()</signal>
   <receiver>singleFileWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>singleFileRadioButton</sender>
   <signal>pressed()</signal>
   <receiver>musicFolderWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>musicFolderRadioButton</sender>
   <signal>pressed()</signal>
   <receiver>singleFileWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>285</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>musicFolderRadioButton</sender>
   <signal>pressed()</signal>
   <receiver>musicFolderWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
